File: vdp-php/tests/vctc/ValidationTest.php
<?php

namespace Vdp;

class ValidationTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->abstractVisaAPIClient = new AbstractVisaAPIClient;
	}
	
	public function testRetreiveListofDecisionRecords() {
		$baseUrl = "vctc/";
		$resourcePath = "validation/v1/decisions/history";
		$queryParams = "?limit=1&page=1";
		$statusCode = $this->abstractVisaAPIClient->doMutualAuthCall ( 'get', $baseUrl.$resourcePath.$queryParams, 'Retrieve List of Recent Decision Records call', '' );
		$this->assertEquals($statusCode, "200");
	}
}

File: vdp-c-sharp/vdp-c-sharp/VDPApiTest/vctc/ValidationTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Vdp
{
    [TestClass]
    public class ValidationTest
    {
        private AbstractVisaAPIClient abstractVisaAPIClient;

        public ValidationTest()
        {
            abstractVisaAPIClient = new AbstractVisaAPIClient();
        }

        [TestMethod]
        public void TestRetreiveListofDecisionRecords()
        {
            string baseUri = "vctc/";
            string resourcePath = "validation/v1/decisions/history";
            string queryString = "?limit=1&page=1";
            string status = abstractVisaAPIClient.DoMutualAuthCall(baseUri + resourcePath + queryString, "GET", "Retreive List of Decision Records Test", "");
            Assert.AreEqual(status, "OK");
        }
    }
}
